.github/workflows/pr.yml

```diff
@@ -83,11 +83,14 @@ jobs:
     steps:
```

Comment:
Should we update the name of this section to make it clear that tests are being run as well?

Reply:
Done